OpenRA.Mods.Common/Traits/Attack/AttackOmni.cs

```diff
@@ -29,6 +29,14 @@ public override Activity GetAttackActivity(Actor self, Target newTarget, bool al
 			return new SetTarget(this, newTarget);
 		}
 
+		public override void ResolveOrder(Actor self, Order order)
```

Comment:
I just wasn't sure if there's a reason why this was not part of `AttackBase` and didn't want to take any risks, but I guess it can't really hurt to move it to `AttackBase`.